src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,35 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class LottoResult {
+    private final List<Rank> ranks;
+
+    public LottoResult() {
+        this.ranks = Arrays.asList(Rank.values());
+    }
+
+    public void plusTicketCount(Rank rankToPlusCount) {
+        for (Rank rank : ranks) {
+            if (rank == rankToPlusCount) {
+                rank.plusTicketCount();
+            }
+        }
+    }
+
+    public long calculateTotalReward() {
+        return ranks.stream()
+                .map(x -> x.getTicketCount() * x.getReward())
+                .reduce(Long::sum)
+                .get();
```

Comment:
optional인데 값이 없을 경우도 처리해야되지 않을까요?
```suggestion
                .orElse(0L);
```